backend/src/main/resources/db/migration/V1_0_7__Add_Users_Personal_Information.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE credentials
+ADD COLUMN name VARCHAR(255);
+
+ALTER TABLE credentials
+ADD COLUMN surname VARCHAR(255);
+
+ALTER TABLE proposals
+DROP COLUMN published;
```

Comment:
This is fine for now but once we are live we cannot remove a column this way. It would not allow us to rollback the deployment and would cause complete data corruption. I can explain in a future meeting if interested.

Reply:
I understand the reasoning not to drop columns but I would love to hear your experience in rolling back databases to a previous state. It was a challenge in the previous projects I worked and I will always be interested in success stories :-)

Reply:
Yes, we would like you to explain us this further, please 🙏